private/pdk-project.ts

```diff
@@ -345,16 +310,12 @@ class PDKRelease extends Release {
       access: "public",
       ...publishConfig,
     });
-  }
-}
 
-/**
- * Utility to deeply clone a value
- * @param value Value to clone
- * @returns Cloned value
- */
-export function cloneDeep(value: any): any {
-  return JSON.parse(JSON.stringify(value));
+    NxProject.of(project)?.addBuildTargetFiles(
```

Comment:
Can't this just be:

`NxProject.of(project)?.addBuildTargetFiles(
      [],
      ["{projectRoot}/LICENSE_THIRD_PARTY"],
      true
    );`

Reply:
That will generate inputs of `["!{projectRoot}/LICENSE_THIRD_PARTY", "!{projectRoot}/LICENSE_THIRD_PARTY/**/*"]`, because we don't know if it is a file or directory. Since we know this is just a file it is best to just be explicit.

NX input seems to require the `/**/*` for directories.

Reply:
How about:

`NxProject.of(project)?.addBuildTargetFiles( [], [File.of("{projectRoot}/LICENSE_THIRD_PARTY")], true );`